client/blocks/taxonomy-manager/list.jsx

```diff
@@ -153,6 +160,11 @@ export class TaxonomyManagerList extends Component {
 		} );
 		const hasDefaultSetting = taxonomy === 'category';
 
+		// add a bogus query arg to trigger a new request on delete
+		if ( this.state.deletedItem ) {
+			query.deletedItem = this.state.deletedItem;
```

Comment:
This is an anti-pattern. We shouldn't update our own props.
What exactly are we trying to achieve by doing this? A re-render?